codegenerator/cli/templates/static/codegen/src/Config.res

```diff
@@ -91,14 +93,16 @@ let make = (
   ~enableRawEvents=false,
   ~entities=[],
 ) => {
-  let events = Js.Dict.empty()
+  let events: eventModLookup = EventLookup.empty()
```

Comment:
When I was refactoring the chain workers, I wanted to change this place to pass `eventModLookup` chain worker functor factories - separately for each chain.

The reasoning is:
- Remove the dependency of chain workers from `Config` - currently the events is the last thing left
- The EventLookup structure is simpler and more optimised, since it's unique for each chain
- It'll allow to have a different implementation of `EventLookup` for Fuel